### PR TITLE
chore: update config

### PR DESCRIPTION
**Pull request by b-wize**
Replaced sway components (workspaces and window) with hyprland

Swapped pulseaudio with wireplumber, added wireplumber module (opens pavucontrol on click)

Addressing some concerns from issue #17

### CODE REVIEW

File: config/files/hyprland/usr/etc/xdg/waybar/config [hunk 1]
```diff
@@ -5,26 +5,26 @@
     // "width": 1280, // Waybar width
     "spacing": 4, // Gaps between modules (4px)
     // Choose the order of the modules
-    "modules-left": ["sway/workspaces", "sway/mode", "sway/scratchpad", "custom/media"],
-    "modules-center": ["sway/window"],
-    "modules-right": ["mpd", "idle_inhibitor", "pulseaudio", "network", "cpu", "memory", "temperature", "keyboard-state", "sway/language", "battery", "battery#bat2", "clock", "tray"],
```
Comment (RoyalOughtness) [inline config/files/hyprland/usr/etc/xdg/waybar/config:10]: please move the entire hyprland/usr/etc/xdg dir from config/files/hyprland to config/files/usr

so this file for example should be in config/files/usr/etc/xdg/waybar/config

File: config/files/hyprland/usr/etc/xdg/waybar/config [hunk 2]
```diff
@@ -5,26 +5,26 @@
...
-    //         "2": "",
-    //         "3": "",
-    //         "4": "",
-    //         "5": "",
-    //         "urgent": "",
-    //         "focused": "",
-    //         "default": ""
-    //     }
-    // },
+    "hyprland/workspaces": {
```
Comment (RoyalOughtness) [inline config/files/hyprland/usr/etc/xdg/waybar/config]: is there a way to make this WM-agnostic?

**Comment by RoyalOughtness** [comment]
please also add a screenshot :)

**Review by RoyalOughtness — CHANGES_REQUESTED** [review]

**Comment by b-wize** [comment]
2024-02-06-131613_hyprshot

This is a current draft. Pardon some of the missing modules. Wireplumber didn't seem to open in the VM, and the tray didn't have anything running with icons lol.

File: config/files/hyprland/usr/etc/xdg/waybar/config (hunk 2 above)
Comment (b-wize) [reply config/files/hyprland/usr/etc/xdg/waybar/config]: Complete. Removed the 'hyprland' portion from workspaces and taskbar to allow the script to feed the value

File: config/files/hyprland/usr/etc/xdg/waybar/config (hunk 1 above)
Comment (b-wize) [reply config/files/hyprland/usr/etc/xdg/waybar/config:10]: Also moved these.

**Comment by RoyalOughtness** [comment]
@b-wize I have no idea why, but for some reason I can't reopen this PR

**Pull request closed without merging**